hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Hangman.Game do
```

Comment:
## +2

I like the way you split that out. The choice of module name is really good, as it gets passed around as the opaque value `game`.